### PR TITLE
phpinfo for sysAdmin panel

### PR DESCRIPTION
**Pull request by kojoty**
@andrixnet , @harrieklomp, @deg-pl 

It would be great if you add me sysAdmin role (access to this sysAdmin panel) at your nodes
My account is kojotyPL for both: OCNL and OCRO

It can be possible that after merge it is needed to reset APC cache or I would like to check details of server configuration from php_info..

### CODE REVIEW

**Comment by harrieklomp** [comment]
@kojoty You are now sysAdmin at OC NL.
BTW i do not see any difference when some one is sysAdmin. I checked on my account.

**Comment by kojoty** [comment]
@harrieklomp Thanks, 

There is the only difference for sysAdmi role: such user can access /sys path like: opencaching.nl/sys
There is APC panel (full access) and after this PR phpinfo...

**Comment by deg-pl** [comment]
@kojoty now you are sysAdmin and advUser on OC UK.
It is a very good idea to publish phpinfo() for sysAdmins!

@andrixnet @harrieklomp Please, add also me ("deg") to sysAdmin role.

File: tpl/stdstyle/sysAdmin/index.tpl.php [hunk 1]
```diff
@@ -2,5 +2,6 @@
 <div>
   <ul>
     <li><h3><a href="/sys/apc">APC</a></h3></li>
+    <li><h3><a href="/sys/phpinfo">PHP-INFO</a></h3></li>
```
Comment (deg-pl) [inline tpl/stdstyle/sysAdmin/index.tpl.php]: Czemu tu nie stosujesz SimpleRouter::getLink?
PS: Brakowało phpinfo w kodzie OC! Dzięki!

**Comment by harrieklomp** [comment]
@deg-pl You are now sysAdmin at OC NL.

File: tpl/stdstyle/sysAdmin/index.tpl.php (hunk 1 above)
Comment (kojoty) [reply tpl/stdstyle/sysAdmin/index.tpl.php]: > Czemu tu nie stosujesz SimpleRouter::getLink?

oczywiście powinno być - tak "na szybko" to dopisałem (czyt. bezmyślnie trochę :))
bo mi właśnie brakowało...

**Pull request merged**